index.html

```diff
@@ -57,6 +57,174 @@ <h2>I`am a Software Developer</h2>
+                  </ul>
+                  <div class="work1-btn ."><button type="button" class="work1-button work-show-button">See Project</button>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="work-display">
+          <div class="work-2">
+            <div class="work2-content">
+              <h1>Profesional Art Printing Data</h1>
+              <h2>Profesional Art Printing Data</h2>
+              <p>First A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry`s standard</p>
+              <ul>
+                <li class="list2">html</li>
+                <li class="list2">bootstrap</li>
+                <li class="list2">ruby </li>
+              </ul>
+            </div>
+            <div class="work2-button"><button type="button">See Project</button></div>
+        </div>
+          <div class="work-2">
+            <div class="work2-content">
+              <h1>Profesional Art Printing Data</h1>
+              <h2>Profesional Art Printing Data</h2>
+              <p>First A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry`s standard</p>
+              <ul>
+                <li class="list2">html</li>
+                <li class="list2">bootstrap</li>
+                <li class="list2">ruby </li>
+              </ul>
+            </div><div class="work2-button"><button type="button">See Project</button></div>
+        </div>
+          <div class="work-2">
+            <div class="work2-content">
+              <h1>Website Protfolio</h1>
+              <h2>Website Protfolio</h2>
+              <p>Third A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry`s standard</p>
+              <ul>
+                <li class="list2">html</li>
+                <li class="list2">bootstrap</li>
+                <li class="list2">ruby </li>
+              </ul>
+            </div>
+            <div class="work2-button"><button type="button">See Project</button></div>
+        </div>
+          <div class="work-2">
+            <div class="work2-content">
+              <h1>Profesional Art Printing Data</h1>
+              <h2>Profesional Art Printing Data</h2>
+              <p>First A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry`s standard</p>
+              <ul><li class="list2">html</li>
+                <li class="list2">bootstrap</li>
+                <li class="list2">ruby </li>
+              </ul>
+            </div>
+            <div class="work2-button"><button type="button">See Project</button></div>
+        </div>
+        <div class="work-2">
+          <div class="work2-content">
+            <h1>Profesional Art Printing Data</h1>
+            <h2>Profesional Art Printing Data</h2>
```

Comment:
- [ ] Kindly follow `HTML` best practices because it will help your website to be more accessible to more people especially those who are visually impaired, following the best practices will help screen readers to navigate such kinds of users without any problem. *h1* tag must be used only one time on one page. so kindly use something else here but make sure that that element should be semantically correct.